cvat-ui/src/actions/auth-actions.ts

```diff
@@ -1,11 +1,11 @@
 // Copyright (C) 2020-2022 Intel Corporation
-// Copyright (C) 2022-2023 CVAT.ai Corporation
+// Copyright (C) 2022-2024 CVAT.ai Corporation
 //
 // SPDX-License-Identifier: MIT
 
 import { ActionUnion, createAction, ThunkAction } from 'utils/redux';
 import { RegisterData } from 'components/register-page/register-form';
-import { getCore } from 'cvat-core-wrapper';
+import { getCore, User } from 'cvat-core-wrapper';
```

Comment:
This file was already refactored in #7431 [here](https://github.com/opencv/cvat/pull/7431/files#diff-90b940843555afade6e77647d5c28b8a61ee191ba992d1e5b6220137c8961694). I dont think we need the changes here

Reply:
#7431 is not merged
When we merge this one, these changes will disappear from another PR.

So, what is the problem?

Reply:
I added it here because now in UserSelector component I need current user id. And, there were not types before